draft-poidt-ccamp-actn-poi-pluggable.md

```diff
@@ -147,42 +166,51 @@ informative:
-   equipment instead of being part of the Optical domain {{my_figure-1}}.
+   New pluggable 
+
+   Coherent pluggable digital coherent optics, such as ZR 
+   {{OIF-400ZR-01-0}} and ZR+ {{Open_ZR-Plus_MSA}}, are enabling new 
+   multilayer network use cases where the DWDM optical transmission
+   function is located within the packet domain equipment instead of 
+   being part of the Optical domain {{my_figure-1}}.
+   This means that an IP/MPLS capable device becomes a multi-technology
+   IP/MPLS/Optical device, as the optical connections (OTSi service
+   and media channels) start and end at such devices.
 
-   ZR and ZR+ (and also CFP2-DCO) deployment in routers has already
-   started and are expanding significantly.  The way the DWDM pluggable
+   The coherent pluggable interface deployment in routers has already
+   started and are expanding significantly.  The way the pluggables 
    are in general managed is not yet completely specified and defined by
    any standard and it is becoming an urgent matter to cover for Service
-   Providers.  Full end-to-end management solution of these DWDM
-   coherent pluggable optics, leveraging on ACTN hierarchical
+   Providers.  A full end-to-end management solution of these pluggable
+   digital coherent optics, leveraging on ACTN hierarchical
    architecture, is becoming critical to allow a wider deployment beyond
-   simple point-to-point high capacity link scenarios between two IP/
+   simple point-to-point high-capacity link scenarios between two IP/
    MPLS routers.
 
-   {{my_figure-0}} The ACTN architecture, defined in {{!RFC8453}}, is used to
-   control the multi-domain network shown in {{my_figure-1}} , where each
-   Packet PNC (P-PNC) is responsible for controlling its IP domain,
-   which can be either an Autonomous System (AS), {{!RFC1930}}, or an IGP
-   area within the same operator network.  Each Optical PNC (O-PNC) in
-   the below topology is responsible for controlling its own Optical
-   Domain.
+   {{my_figure-0}} ACTN architecture, defined in {{!RFC8453}}, is used
+   to control the multi-layer and multi-domain network shown in 
+   {{my_figure-1}}, where each Packet PNC (P-PNC) is responsible for 
+   controlling its IP/MPLS domain, which can be either an Autonomous 
+   System (AS), {{!RFC1930}}, or an IGP area within the same operator 
+   network.  Each Optical PNC (O-PNC) in the below topology is 
+   responsible for controlling its own Optical Domain.
 
 ~~~~ ascii-art
-                           +----------+
-                           |   MDSC   |
-                           +-----+----+
-                                 |
-   MPI interf. +-----------+-----+------+-----------+
+                            +----------+
+                            |   MDSC   |
+                            +-+-++-+-+-+
+                              | |  | |
+               +--------------+ |  | +--------------+
+   MPI interf. |           +----+  +----+           |
```

Comment:
Considering my comment on Domains earlier, the node names in this figure need some adjustment. I propose simple numbering of nodes with no mention of technology/domain.

Reply:
v02